backend/middleware/ensureHttps.js

```diff
@@ -0,0 +1,7 @@
+
+module.exports = function ensureSec (req, res, next) {
+  if (req.headers["x-forwarded-proto"] === "https") {
```

Comment:
This should use https://expressjs.com/en/api.html#req.secure for the detection and probably use https://expressjs.com/en/api.html#req.hostname for the redirect.

Reply:
Cheers Glen